lib/export_import.py

```diff
@@ -115,9 +116,13 @@ def _add_attachments(issue, jira, attachments):
         with BytesIO() as buf:
             for chunk in attachment.iter_content():
                 buf.write(chunk)
-            jira.add_attachment(issue, filename=attachment.filename,
+            jira.add_attachment(issue,
+                    filename=_normalize_filename(attachment.filename),
                     attachment=buf)
 
+def _normalize_filename(value):
+    return unicodedata.normalize('NFKD', value).encode('ascii', 'ignore')
```

Comment:
Hm, interesting - you are using Python 3? I never tried the script with Python 3, but glad it works :). I'll fix this soon and let you know.